app/models/bank_account.rb

```diff
@@ -27,6 +27,10 @@ def import_from_file(file, options = {})
   end
 
   def find_import_method
+    # TODO: Move lookup for import function into plugin
+    return method(:import_from_easybank) if /^AT\d{2}14200\d{11}$/.match(iban)
+    return method(:import_from_sparkasse) if /^AT\d{2}20111\d{11}$/.match(iban)
+    return method(:import_from_holvi) if /^FI\d{2}799779\d{7}[0-9A-Z]$/.match(iban)
```

Comment:
This is really cool :) :)
I do think these specific implementations belongs in a plugin, though: no need to load them for all countries in all instances. One per continent or so? We could enable these plugins by default, but at least there is the option to get rid of them.
What do you think?

Reply:
i would prefer one per country and everything enabled by default